drivers/saul/init_devs/auto_init_hsc.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_auto_init_saul
+ * @{
+ *
+ * @file
+ * @brief       Auto initialization of HSC driver.
+ *
+ * @author      Quang Pham
+ *
+ * @}
+ */
+
+#ifdef MODULE_HSC
+
+#include "assert.h"
+#include "log.h"
+#include "saul_reg.h"
+#include "hsc.h"
+#include "hsc_params.h"
+
+#define ENABLE_DEBUG 1
+#include "debug.h"
```

Comment:
```suggestion
```

It's best to use `LOG_DEBUG()` over `DEBUG()` here, as mixing both mechanisms is confusing and harder to use. E.g. one would have to set both 

```C
#define LOG_LEVEL LOG_DEBUG
```

*and*

```C
#define ENABLE_DEBUG 1
```

As the rest of `auto_init` uses `log.h` for debug output, it would make sense to use it here as well.